server/commands/VolumeCommand.ts

```diff
@@ -2,17 +2,40 @@ import CommandDefinition from '../domain/CommandDefinition';
 import IoConnection from '../clients/IoConnection';
 import VolumeEventMessage from "../domain/event-messages/VolumeEventMessage";
 
+const volChangeTypes: { [s: string]: string; } = {
+  '+': 'increase',
+  '-': 'decrease',
+  default: 'setter',
```

Comment:
`increase` and `decrease` are both verbs. `setter` is not.
```suggestion
  default: 'set',
```